### ISSUE TITLE
Remove Lore

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7892]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7892|
|Type|task|
|Created|2015-05-11 12:55:01Z|
|Branch|https://github.com/twisted/twisted/tree/remove-lore-7892|

Eventually, we should remove Lore completely.

<details><summary>Searchable metadata</summary>

```
trac-id__7892 7892
type__task task
reporter__hawkowl hawkowl
priority__low low
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_remove_lore_7892 branches/remove-lore-7892
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__lore lore
keywords__ 
time__1431348901733697 1431348901733697
changetime__1431788753004514 1431788753004514
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_1"></a>|@hawkowl commented|
|-|-|

(In [44759]) Branching to remove-lore-7892.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

So, in making sure that nothing that anything else uses was inadvertently deprecated, I decided to just do the patch while I didn't have the willpower to hack through anything else.

This probably shouldn't land yet. Maybe in 15.3 or 15.4.

**Comment by twisted-trac** [comment]
|<img alt="adiroiban1's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|adiroiban1 commented|
|-|-|

Changes look good.

From http://twistedmatrix.com/trac/wiki/CompatibilityPolicy

```
At the earliest, release R+2 and one year after release R, but often much later:
```

Lore was deprecated in 14.0.0 (2014-05-08) ... so we are one year after 

We had 15.0 15.1 so we should be past R+2

I think that it can land now ... well in 15.3 as 15.2 is already pre-released.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)<a name="note_4"></a>|@moshez set owner to @hawkowl|
|-|-|

Hi HawkOwl,

Thanks for working on this.

Test failure looks to be the random threading test bug that glyph was cleaning up. Other than that, seems green on the bots. 

The following files still refer to lore (and are not of historic variety or mention its deprecation/removal):
    twisted/python/zsh/_lore (should probably be removed)
    docs/core/howto/book.tex -- \newcommand{\loreref}[1]{% (is book.tex still used by anything non-lore? if not, we can kill it.
                                                            if it is, this command needs a better name)
    docs/core/howto/glossary.rst -- remove lore from glossary?
    docs/core/development/policy/svn-dev.rst -- we should probably update the "build-doc" to mention sphinx
    admin/update-documentation.py -- If this script is still in use, it should blame sphinx and not lore. If not, kill it with fire plz.

The zsh one should definitely be killed as part of this diff. The other ones, I guess, are more of a judgement call whether they have to be done here, but I would certainly prefer it.

If you kill only the zsh file, feel free to merge. Otherwise, please resubmit.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl removed owner|
|-|-|

Thanks for the review, moshez.

1. The zsh one will be covered in [#7898](https://github.com/twisted/twisted/issues/7898).
1. Removed.
1. Removed.
1. Updated to use the proper builder.
1. Killed with fire :D

Builders spun, looking good, please review.

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)<a name="note_6"></a>|@moshez set owner to @hawkowl|
|-|-|

Hi HawkOwl,

Thanks for working on this. Looks good to me -- go ahead and merge!

[Goodbye, lore!]

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_8"></a>|@hawkowl set status to `closed`|
|-|-|

(In [44833]) Merge remove-lore-7892: Remove Lore

Author: hawkowl
Reviewer: adiroiban, moshez
Fixes: [#7892](https://github.com/twisted/twisted/issues/7892)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_9"></a>|@hawkowl set status to `reopened`|
|-|-|

(In [44834]) Revert 'Remove Lore' -- empty directories left over. Reopens: [#7892](https://github.com/twisted/twisted/issues/7892)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_10"></a>|@hawkowl set status to `closed`|
|-|-|

(In [44835]) Merge remove-lore-7892: Remove Lore

Author: hawkowl
Reviewer: adiroiban, moshez
Fixes: [#7892](https://github.com/twisted/twisted/issues/7892)
